editor/svgedit.js

```diff
@@ -15,8 +15,10 @@ svgedit = {
 		XLINK: 'http://www.w3.org/1999/xlink',
 		XML: 'http://www.w3.org/XML/1998/namespace',
 		XMLNS: 'http://www.w3.org/2000/xmlns/' // see http://www.w3.org/TR/REC-xml-names/#xmlReserved
-	}
+	},
+	LAYER_CLASS: 'layer'
 };
+svgedit.LAYER_CLASS_REGEX = new RegExp('(\\s|^)' + svgedit.LAYER_CLASS + '(\\s|$)');
```

Comment:
Not sure where to put these constants. 

Could also be one object directly on _svgedit_ instead of string and regex. Like the following.
```
svgedit = {
  ...
  LAYER: {  CLASS_NAME: 'layer }
};
svgedit.LAYER.CLASS_REGEX = new RegExp('(\\s|^)' + svgedit.LAYER.CLASS_NAME + '(\\s|$)');
```